### PR TITLE
Make menu into sidebar #89

### PR DESCRIPTION
**Pull request by RyoMasumura1201**
This is from issue #89

### CODE REVIEW

File: src/App.module.scss [hunk 1]
```diff
@@ -80,6 +80,13 @@
     }
   }
 
+  .sidebarButton {
+    background-color: white;
```
Comment (diballesteros) [inline src/App.module.scss:86]: Use the css variable var(--header) for the background color of the header button to take in the darkmode colors and var(--contrast) for the font color

File: src/App.tsx [hunk 2]
```diff
@@ -235,9 +227,45 @@ const App: React.FC = () => {
...
+          as={Menu}
+          visible={visible}
+          vertical
+          inverted={appState.darkMode}
+          animation="overlay"
+          onHide={() => setVisible(false)}
+          width="thin"
+          aria-label="options"
+        >
+          <Settings />
```
Comment (diballesteros) [inline src/App.tsx:242]: For <Settings /> <Contact  /> <About /> and <Changelog /> change the trigger to a <Menu.Item /> instead of the <Dropdown.Item />

**Review by diballesteros — CHANGES_REQUESTED** [review]
Everything look's great for the most part!

If you can, please send the updated PR to version/3.4.0. This so all the changes can be batched together for the next version. I'll be sure to update the contributing.md for this to be taken into consideration in future contributions.

I also went ahead and added a slight style change to the hidden file input to avoid the overflow  in the sidebar.

Thank you so much for your contribution!

**Comment by RyoMasumura1201** [comment]
@diballesteros 
Thank you for your review!
I've fixed the points you pointed out, so please check back.

**Comment by diballesteros** [comment]
close #92

**Pull request merged**